docs/reference/datatiers.asciidoc

```diff
@@ -1,100 +1,115 @@
-the <<hot-tier, hot tier>>. User created content prioritises high CPU to support complex
-queries and aggregations in a timely manner, as opposed to the <<hot-tier, hot tier>> which
-prioritises high IO.
-The content data has very long data retention characteristics and from a resiliency perspective
-the indices in this tier should be configured to use one or more replicas.
+* <<content-tier, Content tier>> nodes handle the indexing and query load for a collection 
+of content such as a product catalog.
+* <<hot-tier, Hot tier>> nodes handle the indexing load for time series data such as logs or metrics
+and hold your most recent, most-frequently-accessed data. 
+* <<warm-tier, Warm tier>> nodes hold time series data that is accessed less-frequently 
+and rarely needs to be updated.
+* <<cold-tier, Cold tier>> nodes hold time series data that is accessed occasionally and never updated.
 
-NOTE: new indices that are not part of <<data-streams, data streams>> will be automatically allocated to the
-<<content-tier>>
+When you index documents directly to a specific index, they remain on content tier nodes indefinitely. 
 
-[[hot-tier]]
-==== Hot tier
+When you index documents to a data stream, they initially reside on hot tier nodes. 
+You can configure <<index-lifecycle-management, {ilm}>> ({ilm-init}) policies
+to automatically transition your time series  data through the hot, warm, and cold tiers 
+according to your performance, resiliency and data retention requirements. 
+
+You assign a node to one or more data tiers by setting its <<data-node, data role>> in `elasticsearch.yml`. 
+For example, you might assign your highest-performance nodes to both the hot and content tiers:
 
-The hot tier is made of one or more nodes that have the <<data-hot-node, data_hot>> role.
-It is the {es} entry point for timeseries data. This tier needs to be fast both for reads
-and writes, requiring more hardware resources such as SSD drives. The hot tier is usually
-hosting the data from recent days. From a resiliency perspective the indices in this
+[source,yaml]
+--------------------------------------------------
+node.roles:
+   - data_hot
+   - data_content
+--------------------------------------------------
+
+[discrete]
+[[content-tier]]
+=== Content tier
+
+Data stored in the content tier is generally a collection of things such as a product catalog or article archive.
+Unlike time series data, the value of the content remains relatively constant over time,
+so it doesn't make sense to move it to a tier with different performance characteristics as it ages. 
+Content data typically has long data retention requirements, and you want to be able to retrieve 
+items quickly regardless of how old they are. 
+
+Content tier nodes are usually optimized for query performance--they prioritize processing power over IO throughput
+so they can process complex searches and aggregations and return results quickly.
```

Comment:
Should this be split into multiple sentences? (ie. to avoid the need for "--")

Reply:
I like it as a single sentence since the em dash signifies a relation between the two statements, almost like a colon. It's probably personal preference though :)